Childrens-Social-Care-CPD/wwwroot/stylesheets/custom.css

```diff
@@ -0,0 +1,296 @@
+﻿/* STANDARD COLOURS */
```

Comment:
Is all of this css being used? It looks like, for example, that only the class `.section-card` is being used, and none of the other classes e.g. `.section-card__wborder`. If so, it would be worth removing the unused css. Is that the same for other classes?